test/common/README.md

```diff
@@ -261,6 +261,124 @@ Platform normalizes the `pwd` command.
 
 Synchronous version of `spawnPwd`.
 
+### tagGlue()
+* return [&lt;String>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type)
+
+Tag function for tagged template literals:
+removes all literal line breaks and leading spaces.
+
+<!-- eslint-disable strict -->
+```js
+const assert = require('assert');
+const common = require('../common');
+const { tagGlue } = common;
+
+const unTagged =
+               '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
+
+const tagged = tagGlue`
+  0123456789
+  ABCDEFGHIJKLMNOPQRSTUVWXYZ
+  abcdefghijklmnopqrstuvwxyz
+`;
+
+assert.strictEqual(unTagged, tagged);
+```
+
+### tagUnwrap()
+* return [&lt;String>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type)
+
+Tag function for tagged template literals:
+replaces all literal line breaks and leading spaces with a space, then trims.
+
+<!-- eslint-disable strict -->
+```js
+const assert = require('assert');
+const common = require('../common');
+const { tagUnwrap } = common;
+
+const unTagged =
+   'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod...';
+
+const tagged = tagUnwrap`
+  Lorem ipsum dolor sit amet,
+  consectetur adipiscing elit,
+  sed do eiusmod...
+`;
+
+assert.strictEqual(unTagged, tagged);
+```
+
+### tagLF()
+* return [&lt;String>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type)
+
+Tag function for tagged template literals:
+removes two framing line breaks and only auxiliary indents.
+
+<!-- eslint-disable strict -->
+```js
+const assert = require('assert');
+const common = require('../common');
+const { tagLF } = common;
+
+const unTagged =
+`<!doctype html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title></title>
+  </head>
+  <body>
+    ${process.versions}
+  </body>
+</html>`;
+
+const tagged = tagLF`
+  <!doctype html>
+  <html>
+    <head>
+      <meta charset="UTF-8">
+      <title></title>
+    </head>
+    <body>
+      ${process.versions}
+    </body>
+  </html>
+`;
+
+assert.strictEqual(unTagged, tagged);
+```
+
+### tagCRLF()
```

Comment:
There were other cases in the tests. And potentially can be others (Windows stuff etc).